app/controllers/api/v1/npq_profiles_controller.rb

```diff
@@ -6,7 +6,7 @@ class NPQProfilesController < Api::ApiController
       include ApiTokenAuthenticatable
 
       def create
-        validation_data = NPQValidationData.new(validation_data_params)
+        validation_data = NPQApplication.new(validation_data_params)
```

Comment:
I don't want to split it into multiple models. I just missed that name cause I searched globally for `npq_validation_data`, to avoid the ecf validation data variables :P

Reply:
Possibly not in this PR, but tracing through it, it really cries out to be done at some point.